lib/atlas/initializer_input.rb

```diff
@@ -0,0 +1,23 @@
+module Atlas
+  class InitializerInput
+    include ActiveDocument
+
+    DIRECTORY = 'initializer_inputs'
+
+    attribute :query,           String
+    attribute :share_group,     Symbol
+    attribute :priority,        Integer, default: 0
+    attribute :unit,            String
```

Comment:
The `unit` is not used by ETE (only passed through to clients), so this is not needed here.
Instead, we need the `update_type`, which may be nil, "factor", or "%" here (I hope I am not forgetting one) - there should be a validation to check that it is one of these.